lib/stdlib/doc/src/erl_tar.xml

```diff
@@ -104,21 +105,18 @@
     <title>Limitations</title>
     <list type="bulleted">
       <item>
-        <p>For maximum compatibility, it is safe to archive files with names
-          up to 100 characters in length. Such tar files can generally be
-          extracted by any <c>tar</c> program.</p>
-      </item>
-      <item>
-        <p>For filenames exceeding 100 characters in length, the resulting tar
-          file can only be correctly extracted by a POSIX-compatible <c>tar</c>
+        <p>For filenames exceeding 100 bytes in length (not counting use of the prefix field),
+          the resulting tar file can only be correctly extracted by a USTAR-compatible <c>tar</c>
```

Comment:
The reference to Solaris seems dated.

Are there still any `tar` programs out there that can't handle filenames exceeding 100 characters?